pkg/package-server/server/server.go

```diff
@@ -178,7 +179,9 @@ func (o *PackageServerOptions) Run(stopCh <-chan struct{}) error {
 		return err
 	}
 
-	sourceProvider := provider.NewInMemoryProvider(catsrcSharedIndexInformers, queueOperator, o.GlobalNamespace)
+	// TODO(alecmerdler): Use `operator-registry` as provider instead which watches `CatalogSources` and establishes gRPC connections to each
```

Comment:
I think you can removed these commented lines now?

Reply:
yup